### ISSUE TITLE
Narrator does not read the position number of the equation when user navigates to equation in Tab mode

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1177]
A team member will update this issue with more details soon.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
